src/com/goide/inspections/GoUnusedImportDeclaration.java

```diff
@@ -62,6 +69,19 @@ protected void checkFile(@NotNull GoFile file, @NotNull ProblemsHolder problemsH
     for (GoImportSpec duplicatedImportSpec : GoImportOptimizer.findDuplicatedEntries(importMap)) {
       problemsHolder.registerProblem(duplicatedImportSpec, "Redeclared import", ProblemHighlightType.GENERIC_ERROR, OPTIMIZE_QUICK_FIX);
     }
+
+    // Find imports that are referred to by the same alias or last pathname component.
+    MultiMap<String, GoImportSpec> nameMap = getImportNameMap(file);
```

Comment:
This logic partly duplicates the loop above that is using GoImportOptimizer#findDuplicatedEntries. This leads to duplicated error message
![screenshot 2015-09-09 21 44 35](https://cloud.githubusercontent.com/assets/140920/9771030/1935feba-573c-11e5-91cb-fc1c2373deb2.png)